content/tracing/advanced_usage/manual_tracing.md

```diff
@@ -33,6 +33,41 @@ public class MyClass {
 
 {{% /tab %}}
 {{% tab "Python" %}}
+
+We have a Flask Python application that when called on `/doc` returns **42**.
+We instrumented our python code in order to generate traces from it:
+
+```python
+import time
+import blinker as _
+
+from flask import Flask, Response
```

Comment:
Don't use external libraries (I saw that you imported the example we had). Make this example simple. I'd say to split the example in 2 parts:
* write down a plain function where you use the Python context manager to trace an execution
* write down the same function where you use the `@tracer.wrap()` decorator